src/hardware/disney.cpp

```diff
@@ -27,49 +27,52 @@
 #include "pic.h"
 #include "setup.h"
 
-#define DISNEY_BASE 0x0378
+// Disney Sound Source Constants
+constexpr uint16_t DS_PORT_BASE = 0x0378;
+constexpr uint8_t DS_BUF_SAMPLES = 128;
 
-#define DISNEY_SIZE 128
+constexpr uint8_t DS_ACKNOWLEDGE_BIT = 0b0100'0000;  // 0x40
+constexpr uint8_t DS_INTERRUPT_MASK = 0b1111'1011;   // ~0x4
+constexpr uint8_t DS_INIT_STATUS_BITS = 0b1000'0100; // 0x84
+constexpr uint8_t DS_PARALLEL_IRQ_BIT = 0b0001'0000; // 0x10
+constexpr uint8_t DS_PIN_9_BIT = 0b1000'0000;        // 0x80
```

Comment:
These are introduced, but unused?

Please make up your mind if you want to use binary literals or hex literals :) Binary literals are useful only when working with bitmasks.

Reply:
I can leave them as hex for now.
Maybe if we can define all of the magic numbers, the connections between those individual bits versus the masks will make it worth while to visualize as bit fields.

(Given the mask might either be a handful off bit-states turned on or turned off; or might be the opposite of a single bit state).

Reply:
> These are introduced, but unused?

They used in a subsequent commit (due to the granularity problem I had with git add -p). When I redo the PR, I will be sure to introduce them with the code that uses them.